ephios/user_management/forms.py

```diff
@@ -114,6 +126,36 @@ class Meta:
         model = Group
         fields = ["name"]
 
+    def __init__(self, **kwargs):
+        if (group := kwargs.get("instance", None)) is not None:
+            kwargs["initial"] = {
+                "users": group.user_set.all(),
+                "can_view_past_event": group.permissions.filter(
+                    codename="view_past_event"
+                ).exists(),
+                "can_add_event": group.permissions.filter(codename="add_event").exists(),
+                "publish_event_for_group": get_objects_for_group(
+                    group, "publish_event_for_group", klass=Group
+                ),
+                "can_manage_user": group.permissions.filter(
+                    codename__in=[
+                        "add_userprofile",
+                        "change_userprofile",
+                        "delete_userprofile",
+                        "view_userprofile",
+                    ]
+                ).exists(),
+                "can_manage_group": group.permissions.filter(
+                    codename__in=[
+                        "add_group",
+                        "change_group",
+                        "delete_group",
+                        "view_group",
+                    ]
+                ).exists(),
```

Comment:
```suggestion
                ).exists(),
                **kwargs.get("initial", {}),
```

or smth. like that. Has the disadvantage that the `exists` queries here are run even if not used, but I did the same for EventForm and avoiding that would need more complex control flow.